### ISSUE TITLE
scroll events not emitted 

### ISSUE THREAD

**Issue by spaceface777** [kovidgoyal/kitty#3094]
**Describe the bug**
Mouse scroll events don't seem to be emitted. Other mouse events do work properly.

**To Reproduce**
1. `echo -e "\e[?1003h\e[?1015h\e[?1006h"`

**Expected behavior**
I expected to get events along the lines of `\e[<64;xx;yyM`, where xx and yy are the scroll position X and Y respectively, and 64 is the event code for scrolling downwards, and 65 for scrolling upwards

**Environment details**
OS: pop-OS 20.10, x11

```
Output of kitty --debug-config
kitty 0.18.3 created by Kovid Goyal
Linux pop-os 5.8.0-7625-generic #26~1604441477~20.10~d41e407-Ubuntu SMP Wed Nov 4 01:25:00 UTC 2 x86_64
Pop!_OS 20.10 \n \l
DISTRIB_ID=Pop
DISTRIB_RELEASE=20.10
DISTRIB_CODENAME=groovy
DISTRIB_DESCRIPTION="Pop!_OS 20.10"
Loaded config files: /etc/xdg/kitty/kitty.conf, /home/spaceface/.config/kitty/kitty.conf
Running under: X11

Config options different from defaults:
adjust_line_height    2
allow_remote_control  y
background            Color(red=34, green=34, blue=34)
background_opacity    0.9
font_family           JetBrains Mono
font_size             13.0
update_check_interval 0.0
```

**Additional context**
- also ocurrs with `kitty --config NONE`
- gnome-terminal emits these events properly

**Comment by spaceface777** [comment]
hm, it seems like this works when using the latest binary, not the release on apt. Sorry for the noise, and keep up the great work! :slightly_smiling_face:

**Comment by trygveaa** [comment]
See 0bff30e9544366398f247788f133690f0c7c2f3f.

**Comment by spaceface777** [comment]
Cool, thanks for the info.

**Comment by spaceface777** [comment]
I looked at #2979 multiple times before opening this issue when searching for duplicates, but I couldn't find any direct references to that commit and/or the 0.19.0 changelogs on there. Perhaps it would be a good idea to link it inside that issue.

**Comment by trygveaa** [comment]
Did you mean #2939? I can't see any relevant info in #2979.

That commit is referenced in #2939:

> kovidgoyal closed this in 0bff30e on Aug 27
